test/test.el

```diff
@@ -533,6 +533,32 @@ buffer and do not save the results."
            :groups '((:pred (lambda (item)
                               (s-contains? "moon" item)))))))
 
+(ert-deftest org-super-agenda--test-:property-list-val ()
+  ;; DONE: Works.
+  (should (org-super-agenda--test-run
+           :groups '((:property ("Effort" "5" ))))))
```

Comment:
Nitpick: extra space char.  :)

Reply:
Well spotted! Also note to self to tend to my elisp development setup - it should have highlighted it.